### ISSUE TITLE
SelectButton: aria-disabled contains code instead of a boolean value

### ISSUE THREAD

**Issue by midyro** [primefaces/primevue#5235]
### Describe the bug

While using the SelectButton component with the option-disabled, I saw in that the  `aria-disabled` attribute wasn't a boolean but the code to disable the option.

If you open inspect, the HTML looks like this:
```html
<div class="p-selectbutton p-buttonset p-component" role="group" data-pc-name="selectbutton" data-pc-section="root" name="demo"><div tabindex="0" aria-label="Test value 1" role="radio" aria-checked="true" aria-disabled="(option) => {
  if (option.value === 2) {
    return true;
  }

  return false;
}" ...
```

### Reproducer

https://stackblitz.com/edit/primevue-create-vue-issue-template-4k4vwi

### PrimeVue version

3.4.8

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vite

### Steps to reproduce the behavior

1. Create a component that uses a SelectButton
2. Add a few options
3. Add the prop `option-disabled` with either a function or a string as the value
4. Use the browsers inspect function and check the code of the SelectButton

### Expected behavior

The `aria-disabled` attribute should be a boolean.
